### ISSUE TITLE
Narrator/NVDA does not announce about selection status of "helpful" and "Unhelpful" button present within the IRIS reply card

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5421]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen Reader users are impacted as they might not know about the selection status of the "Helpful"/"unhelpful" button.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### What is the public URL for the website?

Intelligent Support - Iris Copilot

### How to reproduce the issue?

1.	Turn on Narrator/NVDA.
2.	Launch "Iris-PPE" Application on "Microsoft Teams".
3.	Tab to "Type a message" edit field and type "how to access esxp" and press enter.
4.	In Response, App will send a card.
5.	Tab to the card and press enter to navigate through the card elements.
6.	Tab to "helpful"/"unhelpful" element present within the card.
7.	Verify Narrator/NVDA announce selected/unselect status for "helpful"/"unhelpful" button.

### What do you expect?

Narrator/NVDA should announce about selected/unselected status of "helpful" and "Unhelpful" button present within the IRIS reply card.

### What actually happened?

Narrator/NVDA does not announce about selection status of "helpful" and "Unhelpful" button present within the IRIS reply card.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/53847e7e-018c-47b6-8f51-a4a0f0c14794

### MAS reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Adaptive Card JSON

```json

```

### Additional context

Bug copied from Intelligent Support - Iris Copilot and please find comments below.

As per PO latest comment “This is need to validated against Bot framework support. if Adaptive card 1.6 fixed that particular version needs to be supported in Bot Framework to start consuming this fix. Please work with bot framework for supported version”

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#MAS1.3.1;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@compulim We are following up on the bug fix, could you please check and confirm when the fix will be available?

**Comment by OEvgeny** [comment]
This is not Web Chat experience. This is Teams
